generatebundlefile/hack/release.sh

```diff
@@ -92,6 +87,10 @@ function push () {
     "$ORAS_BIN" push "${REPO}:v${version}-latest" bundle.yaml
 }
 
+docker logout public.ecr.aws
+export AWS_PROFILE=prod
+aws ecr-public get-login-password --region us-east-1 | HELM_EXPERIMENTAL_OCI=1 helm registry login --username AWS --password-stdin public.ecr.aws
```

Comment:
this is all fine, but we should have used HELM_REGISTRY_CONFIG=~/whereever/this/file/is